scripts/router-dashboard-api-wrapper.py

```
@@ -0,0 +1,298 @@
+                "rx_errors": int(self.read_file(stats_path / "rx_errors") or 0),
+                "tx_errors": int(self.read_file(stats_path / "tx_errors") or 0),
+            }
+
+        self.send_json(interfaces)
+    except Exception as exc:
+        self.send_error_json(500, str(exc))
+
+
+def handle_firewall_stats(self):
+    try:
+        result = subprocess.run(
```

Ruff (0.15.9)

[error] Undefined name `subprocess`

(F821)